index.html

```diff
@@ -63,5 +63,143 @@ <h2 class="let">LET'S CONNECT</h2>
       
     </div>
     </section>
+    <section class="works">
+      <div class="card">
+        <div class="card__image">
+          <img src="images/tox1.png" alt="">
+        </div>
+        <div class="card__content">
+          <h2 class="card__content--heading">Tonic</h2>
+          <ul class="card__content--item1">
+            <li class="card__item--heading"><img src="" alt=""> Canopy</li>
+            <li> <img src="images/Counter.png" alt=""> Back End Dev</li>
+            <li> <img src="images/Counter.png" alt=""> 2015</li>
+          </ul>
+          <p class="card__content--desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="card__content--item2">
+            <li class="tech__btn">html</li>
+            <li class="tech__btn">css</li>
+            <li class="tech__btn">Javascript</li>
+          </ul>
+          <button class="btn">See Project</button>
+        </div>
+      </div>
+      
+      <div class="card">
+        <div class="card__image">
+          <img src="images/mul2.png" alt="">
+        </div>
+        <div class="card__content">
+          <h2 class="card__content--heading">Multi-Post Stories</h2>
+          <ul class="card__content--item1">
+            <li class="card__item--heading"><img src="" alt=""> Canopy</li>
+            <li> <img src="images/Counter.png" alt=""> Back End Dev</li>
+            <li> <img src="images/Counter.png" alt=""> 2015</li>
+          </ul>
+          <p class="card__content--desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="card__content--item2">
+            <li class="tech__btn">html</li>
+            <li class="tech__btn">css</li>
+            <li class="tech__btn">Javascript</li>
+          </ul>
+          <button class="btn">See Project</button>
+        </div>
+      </div>
+      <div class="card">
+        <div class="card__image">
+          <img src="images/ton21.png" alt="">
+        </div>
+        <div class="card__content">
+          <h2 class="card__content--heading">Tonic</h2>
+          <ul class="card__content--item1">
+            <li class="card__item--heading"><img src="" alt=""> Canopy</li>
+            <li> <img src="images/Counter.png" alt=""> Back End Dev</li>
+            <li> <img src="images/Counter.png" alt=""> 2015</li>
+          </ul>
+          <p class="card__content--desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="card__content--item2">
+            <li class="tech__btn">html</li>
+            <li class="tech__btn">css</li>
+            <li class="tech__btn">Javascript</li>
+          </ul>
+          <button class="btn">See Project</button>
```

Comment:
- [ ] Please kindly add `button `interaction for all your `button `elements for great user experience. 👍